src/Domain.LinnApps/Parts/MechPartSourceService.cs

```diff
@@ -63,14 +58,161 @@ public string GetCapacitanceLetterAndNumeralCode(string unit, decimal value)
+            current.CapacitorVoltageRating = updated.CapacitorVoltageRating;
+            current.CapacitorPositiveTolerance = updated.CapacitorPositiveTolerance;
+            current.CapacitorNegativeTolerance = updated.CapacitorNegativeTolerance;
+            current.CapacitorDielectric = updated.CapacitorDielectric;
+            current.Package = updated.Package;
+            current.CapacitorPitch = updated.CapacitorPitch;
+            current.CapacitorLength = updated.CapacitorLength;
+            current.CapacitorWidth = updated.CapacitorWidth;
+            current.CapacitorHeight = updated.CapacitorHeight;
+            current.CapacitorDiameter = updated.CapacitorDiameter;
+            current.CapacitanceLetterAndNumeralCode = updated.CapacitanceLetterAndNumeralCode;
+            current.Resistance = updated.Resistance;
+            current.ResistorTolerance = updated.ResistorTolerance;
+            current.Construction = updated.Construction;
+            current.ResistorLength = updated.ResistorLength;
+            current.ResistorHeight = updated.ResistorHeight;
+            current.ResistorWidth = updated.ResistorWidth;
+            current.ResistorPowerRating = updated.ResistorPowerRating;
+            current.ResistorVoltageRating = updated.ResistorVoltageRating;
+            current.TemperatureCoefficient = updated.TemperatureCoefficient;
+            current.TransistorType = updated.TransistorType;
+            current.TransistorDeviceName = updated.TransistorDeviceName;
+            current.TransistorCurrent = updated.TransistorCurrent;
+            current.TransistorVoltage = updated.TransistorVoltage;
+            current.TransistorPolarity = updated.TransistorPolarity;
+            current.IcType = updated.IcType;
+            current.IcFunction = updated.IcFunction;
+            current.LibraryRef = updated.LibraryRef;
+            current.FootprintRef = updated.FootprintRef;
+            current.RkmCode = updated.RkmCode;
+            current.ApplyTCodeBy = updated.ApplyTCodeBy;
+            current.ApplyTCodeDate = updated.ApplyTCodeDate;
+            current.CancelledBy = updated.CancelledBy;
+            current.DateCancelled = updated.DateCancelled;
+            current.McitVerifiedBy = updated.McitVerifiedBy;
+            current.McitVerifiedDate = updated.McitVerifiedDate;
+            current.QualityVerifiedBy = updated.QualityVerifiedBy;
+            current.QualityVerifiedDate = updated.QualityVerifiedDate;
+            current.RemoveTCodeBy = updated.RemoveTCodeBy;
+            current.RemoveTCodeDate = updated.RemoveTCodeDate;
+            current.VerifiedBy = updated.VerifiedBy;
+            current.VerifiedDate = updated.VerifiedDate;
+            current.PurchasingQuotes = updated.PurchasingQuotes;
+            current.Usages = updated.Usages;
+            current.LifeExpectancyPart = updated.LifeExpectancyPart;
+            current.Configuration = updated.Configuration;
```

Comment:
They might already be somewhere but are there tests for these million bits of the MechPartSources to check they update the right thing? Looks fine to my eye but there's just so many

Reply:
No.. there are tests for anything thats more complicated than just a this.that = that.that but you'r e right, I should add tests for all of this incredibly boring code